### PR TITLE
Do not merge -- Dice on table

### PR DESCRIPTION
**Pull request by sethalves**
some physics/network related debugging

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: c1d627be4fa214b0ed7195fb10457de8747920f9 available at: https://gist.github.com/c52cf6ff368f84053405<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.dmg

File: libraries/entities/src/EntityItem.cpp [hunk 1]
```diff
@@ -416,22 +416,22 @@ int EntityItem::readEntityDataFromBuffer(const unsigned char* data, int bytesLef
...
-            }
-        } else {
-            // If this isn't from the same sever packet, then honor our skew adjusted times...
-            // If we've changed our local tree more recently than the new data from this packet
-            // then we will not be changing our values, instead we just read and skip the data
-            if (_lastEdited > lastEditedFromBufferAdjusted) {
-                ignoreServerPacket = true;
-            }
-        }
+        // // If this packet is from the same server edit as the last packet we accepted from the server
```
Comment (ZappoMan) [inline libraries/entities/src/EntityItem.cpp]: maybe you can disable this with a #ifdef instead?

**Comment by hifi-gustavo** [comment]
Build console output for commit: 17b34a518cb5d08e39e932b5922c22add7011197 available at: https://gist.github.com/c52cf6ff368f84053405<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: f4532f7f1be53234cfee17ed79de71ce4f33cf6b available at: https://gist.github.com/c52cf6ff368f84053405<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.dmg

File: libraries/entities/src/EntityItem.cpp (hunk 1 above)
Comment (sethalves) [inline libraries/entities/src/EntityItem.cpp]: I don't really plan to merge this branch.

**Comment by hifi-gustavo** [comment]
Build console output for commit: b7bacc4c5e9e5640acdc420b0e31a0fed14bbf69 available at: https://gist.github.com/c52cf6ff368f84053405<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: fda3e6ab3e4edfe8b8cadd28e68b436c0e8c6bb7 available at: https://gist.github.com/c52cf6ff368f84053405<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.dmg

**Comment by hifi-gustavo** [comment]
Build console output for commit: 9100dd5fafb0ccb4c371655738100abc6d032a41 available at: https://gist.github.com/c52cf6ff368f84053405<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4766.dmg

**Pull request closed without merging**